### ISSUE TITLE
FaceDetector is not exported in 0.10.5

### ISSUE THREAD

**Issue by satoren** [google/mediapipe#4799]
### OS Platform and Distribution

web

### Programming Language and version

Javascript

### MediaPipe version

0.10.5

### Android AAR (if android)

None

### Describe the problem
open
https://codepen.io/mediapipe-preview/pen/OJByWQr?editors=1111
and change version to `tasks-vision@0.10.5`

### Complete Logs

```shell
Uncaught SyntaxError: The requested module 'https://cdn.jsdelivr.net/npm/@mediapipe/tasks-vision@0.10.5' does not provide an export named 'FaceDetector' 
 at https://cdpn.io/cpe/boomboom/pen.js?key=pen.js-73446071-a9ea-9a47-118d-dda5b5e8d1b4:11
```

**Comment by schmidt-sebastian** [comment]
Fix pending internal review. Sorry about that.

**Comment by schmidt-sebastian** [comment]
Released 0.10.6

**Comment by YashPotdar** [comment]
It looks like it's the same for Python. FaceDetector (face_detector.py) is in Github, but not local when running `pip install mediapipe`.
